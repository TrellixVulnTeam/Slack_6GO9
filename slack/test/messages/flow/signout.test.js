const { SignOut } = require('../../../lib/messages/flow');

describe('SignOut rendering', () => {
  test('works', async () => {
    const message = new SignOut('https://example.org/github/oauth/login?state=abcdef', 'U012345');
    expect(message.toJSON()).toMatchSnapshot();
  });
});
