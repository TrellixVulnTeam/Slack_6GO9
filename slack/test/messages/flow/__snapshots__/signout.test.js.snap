// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignOut rendering works 1`] = `
Object {
  "attachments": Array [
    Object {
      "actions": Array [
        Object {
          "style": "primary",
          "text": "Connect GitHub account",
          "type": "button",
          "url": "https://example.org/github/oauth/login?state=abcdef",
        },
      ],
      "color": "#24292f",
      "fallback": "You have signed out from GitHub.",
      "mrkdwn_in": Array [
        "text",
      ],
      "text": ":white_check_mark: <@U012345> is now signed out
Features like subscriptions and rich link previews will stop working. Use \`/github signin\` to sign back into your GitHub account at any time.",
    },
  ],
  "response_type": "ephemeral",
}
`;
